pocs/utils/logger.py

```diff
@@ -8,6 +8,33 @@
 from .config import load_config
 
 
+class PanLogger(object):
+    """ Logger for PANOPTES with format style strings """
+
+    def __init__(self, logger):
+        super(PanLogger, self).__init__()
+        self.logger = logger
+
+    def _process_str(self, fmt, *args, **kwargs):
+        log_str = fmt
+        if len(args) > 0:
+            log_str = fmt.format(*args, **kwargs)
+
+        return log_str
+
+    def debug(self, fmt, *args, **kwargs):
+        self.logger.debug(self._process_str(fmt, *args, **kwargs))
+
+    def info(self, fmt, *args, **kwargs):
+        self.logger.info(self._process_str(fmt, *args, **kwargs))
+
+    def warning(self, fmt, *args, **kwargs):
```

Comment:
warn rather than warning? Or change the one use of warning into warn?

Reply:
I think you mean the other way around? I changed the one case of `warn` to `warning`

Reply:
OK, which ever way it is supposed to be.